schema-engine/sql-introspection-tests/tests/cockroachdb/constraints.rs

```diff
@@ -0,0 +1,142 @@
+//! https://www.notion.so/prismaio/PostgreSQL-Exclusion-Constraints-fb2ecc44f773463f908d3d0e2d737271
+
+use indoc::indoc;
+use sql_introspection_tests::test_api::*;
+use test_macros::test_connector;
+
+#[test_connector(tags(CockroachDb))]
+async fn aragon_test_cockroachdb(api: &mut TestApi) -> TestResult {
+    let raw_sql = indoc! {r#"
+        CREATE TABLE users (
+            user_id INT8 PRIMARY KEY
+        );
```

Comment:
This test was originally in `schema-engine/sql-introspection-tests/tests/postgres/constraints.rs`. Moved it here, as it's CockroachDB-only.